this package doesn't compile

Hi,
I'm having problems using this trigger as dependency in other project
also this same project doesn't seems to build to me.. at least using
those build instruction 
https://github.com/spring-cloud-stream-app-starters/trigger/blob/master/spring-cloud-starter-stream-source-trigger/README.adoc

( tested on both master && 1.3.x branch ) 
`$ ./mvnw clean install -PgenerateApps`
give this output
```
[INFO] Compiling 1 source file to /tmp/trigger/spring-cloud-starter-stream-source-trigger/target/test-classes
[INFO] -------------------------------------------------------------
[ERROR] COMPILATION ERROR : 
[INFO] -------------------------------------------------------------
[ERROR] /tmp/trigger/spring-cloud-starter-stream-source-trigger/src/test/java/org/springframework/cloud/stream/app/trigger/source/TriggerSourceTests.java:[29,52] package org.springframework.cloud.stream.test.binder does not exist
[ERROR] /tmp/trigger/spring-cloud-starter-stream-source-trigger/src/test/java/org/springframework/cloud/stream/app/trigger/source/TriggerSourceTests.java:[48,19] cannot find symbol
  symbol:   class MessageCollector
  location: class org.springframework.cloud.stream.app.trigger.source.TriggerSourceTests
```

--- a/spring-cloud-starter-stream-source-trigger/pom.xml
+++ b/spring-cloud-starter-stream-source-trigger/pom.xml
@@ -26,6 +26,11 @@
 			<groupId>org.springframework.cloud.stream.app</groupId>
 			<artifactId>app-starters-annotation-common</artifactId>
 		</dependency>
+		<dependency>
+			<groupId>org.springframework.cloud</groupId>
+			<artifactId>spring-cloud-stream-test-support</artifactId>
+			<scope>test</scope>
+		</dependency>
 	</dependencies>
 
 	<build>

--- a/trigger-app-dependencies/pom.xml
+++ b/trigger-app-dependencies/pom.xml
@@ -1,30 +1,31 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
-    <modelVersion>4.0.0</modelVersion>
-    <groupId>org.springframework.cloud.stream.app</groupId>
-    <artifactId>trigger-app-dependencies</artifactId>
-    <version>2.0.0.BUILD-SNAPSHOT</version>
-    <packaging>pom</packaging>
-    <name>trigger-app-dependencies</name>
-    <description>Spring Cloud Stream Trigger App Dependencies</description>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+		 xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+	<groupId>org.springframework.cloud.stream.app</groupId>
+	<artifactId>trigger-app-dependencies</artifactId>
+	<version>2.0.0.BUILD-SNAPSHOT</version>
+	<packaging>pom</packaging>
+	<name>trigger-app-dependencies</name>
+	<description>Spring Cloud Stream Trigger App Dependencies</description>
 
-    <parent>
-	<artifactId>spring-cloud-dependencies-parent</artifactId>
-        <groupId>org.springframework.cloud</groupId>
-        <version>2.0.0.BUILD-SNAPSHOT</version>
-        <relativePath />
-    </parent>
+	<parent>
+		<artifactId>spring-cloud-dependencies-parent</artifactId>
+		<groupId>org.springframework.cloud</groupId>
+		<version>2.0.0.BUILD-SNAPSHOT</version>
+		<relativePath/>
+	</parent>
 
-    <dependencyManagement>
-        <dependencies>
-            <dependency>
-                <groupId>org.springframework.cloud.stream.app</groupId>
-                <artifactId>spring-cloud-starter-stream-source-trigger</artifactId>
-                <version>2.0.0.BUILD-SNAPSHOT</version>
-            </dependency>
-        </dependencies>
-    </dependencyManagement>
-<profiles>
+	<dependencyManagement>
+		<dependencies>
+			<dependency>
+				<groupId>org.springframework.cloud.stream.app</groupId>
+				<artifactId>spring-cloud-starter-stream-source-trigger</artifactId>
+				<version>2.0.0.BUILD-SNAPSHOT</version>
+			</dependency>
+		</dependencies>
+	</dependencyManagement>
+	<profiles>
 		<profile>
 			<id>spring</id>
 			<repositories>

